spinnman/connections/udp_packet_connections/udp_connection.py

```diff
@@ -39,6 +39,8 @@ def __init__(self, local_host=None, local_port=None, remote_host=None,
 
             # Create a UDP Socket
             self._socket = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+            self._socket.setsockopt(
+                socket.SOL_SOCKET, socket.SO_RCVBUF, 1048576)
```

Comment:
I recommend putting the call to `setsockopt` in a try/catch/warn as a failure to set the receive buffer size (for whatever weird system reason moseys along) isn't fatal but might make things not work too well.